index.html

```diff
@@ -2,73 +2,76 @@
 layout: default
 ---
 
-<div class="header-container jumbotron">
+<!-- Place this tag in your head or just before your close body tag. -->
+<script async defer src="https://buttons.github.io/buttons.js"></script>
+
+<div class="header-container bg-primary jumbotron">
     <div class="container">
-        <h1>Language Server Protocol</h1>
-        <p>
+        <div class="intro">
+            <img src="{{site.baseurl}}/img/logo.svg" width="150">
+            <h1>Language Server Protocol</h1>
+        </div>
+        <p class="intro-text">
             The Language Server Protocol (LSP) defines the protocol used between an editor or IDE to talk to a language server that provides language features like auto complete, goto definition, find all references.
         </p>
+
+        <br />
+        <div class="intro-text">
+            <!-- Place this tag where you want the button to render. -->
+            <a class="github-button" href="https://github.com/Microsoft/language-server-protocol/subscription" data-size="large" aria-label="Watch Microsoft/language-server-protocol on GitHub">Watch</a>
+        </div>
     </div>
 </div>
 
 <div class="container">
     <div class="row align-items-center">
-        <div class="col-lg-6">
+        <div class="col-lg-7">
             <h2 class="header-light regular-pad">What is the Language Server Protocol?</h2>
-            <blockquote>
-              <p>Implementing language support like auto complete, goto definition, or type hovering for a programming language for different development tools is a significant effort. Each development tool provides different APIs for language extenders. Moreover, an extension has to be implemented in the implementation language of the tool itself. This requires to implement the smartness for a language multiple times. The idea behind a <i>Language Server</i> is to provide the language smarts inside a server and using inter process communication and a protocol to talk to this server. If each language server speaks a different protocol then the  integration of each server into each tool is custom. The idea behind the <i>Language Server Protocol</i> is to standardize this protocol and therefore to simplify the integration of different language servers in a development tool. This is a win for both the language provider and the development tool provider!
-              </p>
-            </blockquote>
+
+            <p>Implementing support for features like autocomplete, goto definition, or documentation on hover for a programming language is a significant effort. Traditionally this work must be repeated for each development tool, as each provides different APIs for implementing the same features.</p>
+            <p>The idea behind a Language Server is to provide the language-specific smarts inside a server that can communicate with development tooling inter-process communication through a protocol.</p>
```

Comment:
Ending this sentence with `through a protocol` is a tad unnecessary given that the next sentence introduces the protocol, and since all IPC communication is necessarily through a protocol, if perhaps an ad-hoc one.